configuration/config-txt/video.md

```diff
@@ -336,6 +336,38 @@ Note that this simply **creates** the mode (group 2 mode 87). In order to make t
 
 This may not work if your monitor does not support standard CVT timings.
 
+### LCD display/touchscreen options
+
+#### ignore_lcd
+
+By default no DSI LCD detection is run (`ignore_lcd=1`). To enable detection (and accordingly the LCD display), use `ignore_lcd=0`
+
+#### display_default_lcd
+
+Specify which display is to be used as the default. To set the LCD as the default, use `display_default_lcd=1`; to use HDMI as the default, use `display_default_lcd=0`
```

Comment:
If a DSI LCD is detected it will be the default display (showing the framebuffer). Set ``display_default_lcd=0`` to not have the LCD as default (it can still be used by choosing the display number from supported apps like omxplayer).